GBHFacebookImagePicker/Classes/Model/GBHFacebookPickerConfig.swift

```diff
@@ -38,6 +38,12 @@ public struct GBHFacebookPickerConfig {
 
         /// Place checkView
         public var placeCheckView: CheckViewPosition = .bottomRight
+
+        /// Selected border color
+        public var selectedBorderColor: UIColor?
+
+        /// Selected border width
+        public var selectedBorderWidth: CGFloat = 0
```

Comment:
Can you please update the README / exemple with this two settings ? :-)

Reply:
I think the README already include these settings: 

// Selected border color 
GBHFacebookImagePicker.pickerConfig.uiConfig.selectedBorderColor = UIColor.red

// Selected border width 
GBHFacebookImagePicker.pickerConfig.uiConfig.selectedBorderWidth = 4.0